docs/disabled_features.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Disabled Features
+sort_rank: 10
+---
+
+# Disabled Features
+
+Here is a list of features that are disabled by default since they are breaking changes.
+They can be enabled using the `--enable-feature` flag and will be enabled by default
```

Comment:
The comma separated should be mentioned here. It's not sufficient that they will be enabled if they're widely adopted, some also need to wait for a major version. I'd hedge things by saying something like may be enabled by default in a future version.

Reply:
Looks like comma separated value is not working, we need something like `--enable-feature=promql-at-modifier --enable-feature=something-more`. I will make it work for comma separated.

Reply:
Instead of saying they will be eabled by default we should explicitly say that they are considered experimental.

Reply:
The feature is actually stable and not enabled by default since it's breaking. I would not consider that as experimental.

Reply:
In general I would like we mark all these as experimental.

In the current case, the consensus was:

  As an experimental feature behind a feature flag, we will allow query evaluation to access samples with a timestamp larger than the evaluation timestamp

Reply:
That slipped my mind, yes it should be experimental in the sense that we are allowed to change the behavior if we want

Reply:
I have updated it now, mentioning comma separated values, experimental, and may be enabled by default.